package.json

```diff
@@ -33,10 +33,10 @@
 		"yargs": "^17.0.1"
 	},
 	"scripts": {
-		"build": "npx tsc --build . spec && node scripts/parser-compiler.js",
-		"watch": "npx tsc --build . spec --watch",
-		"test": "node ./spec/build/spec.js",
-		"lint": "npx eslint src/**/*.ts spec/src/**/*.ts"
+		"build": "tsc && node scripts/parser-compiler.js",
+		"watch": "tsc -w",
```

Comment:
This assumes that the user has typescript globally installed. And that it is of correct version.

Reply:
"npm comes bundled with npx (since v5.2.0)"